apps/web-client/src/lib/db/cr-sqlite/__tests__/reconciliation-order.test.ts

```diff
@@ -195,4 +195,33 @@ describe("Suppliers order creation", () => {
 			);
 		});
 	});
+
+	describe("Reconciliation order finalization", () => {
+		let db: DB;
```

Comment:
The line `let db: DB` and the first line in `beforeEach` (`db = await getRandomDb();`) can be removed: the `describe("Reconciliation order finalization", ...)` directive is nested under `describe("Suppliers order creation", ...)`, so it inherits its `beforeEach` that already takes care of db initialization.